pkg/capi2mapi/machine.go

```diff
@@ -27,34 +30,74 @@ const (
 	workerUserDataSecretName = "worker-user-data"
 )
 
-// fromMAPIMachineToCAPIMachine translates a MAPI Machine to its Core CAPI Machine correspondent.
-func fromMAPIMachineToCAPIMachine(capiMachine *capiv1.Machine) *mapiv1.Machine {
+// fromCAPIMachineToMAPIMachine translates a core CAPI Machine to its MAPI Machine correspondent.
+//
+//nolint:funlen
+func fromCAPIMachineToMAPIMachine(capiMachine *capiv1.Machine) (*mapiv1.Machine, error) {
+	errs := []*field.Error{}
+
 	mapiMachine := &mapiv1.Machine{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:        capiMachine.Name,
 			Namespace:   mapiNamespace,
 			Labels:      capiMachine.Labels,
 			Annotations: capiMachine.Annotations,
+			// OwnerReferences: These need to be converted so that any MachineSet owning a Machine is represented with the correct owner reference between the two APIs.
```

Comment:
Does this need a jira card reference? The comment confuses me because it says a conversion is needed, but there's no logic or card reference.